src/components/Popover/index.ts

```diff
@@ -0,0 +1,21 @@
+import { Content } from "./Content";
+import { Anchor } from "./Anchor";
+import { Portal } from "./Portal";
+import { Trigger } from "./Trigger";
+import { Arrow } from "./Arrow";
+import { PopoverRoot } from "./Root";
+
+export type { PopoverContentProps } from "./Content";
+export type { PopoverAnchorProps } from "./Anchor";
+export type { PopoverPortalProps } from "./Portal";
+export type { PopoverTriggerProps } from "./Trigger";
+export type { PopoverArrowProps } from "./Arrow";
+export type { PopoverProps } from "./Root";
+
+export const Popover = Object.assign(PopoverRoot, {
+  Trigger,
+  Content,
+  Anchor,
+  Portal,
```

Comment:
**issue**: Any reason to expose Portal as aa part of Popover? I think if we want to expose it, it can be a separated component

Reply:
I can't think of a good use case, so I will remove it.